modules/@apostrophecms/http/ui/apos/big-upload-client.js

```diff
@@ -82,7 +81,9 @@ export default async (url, options) => {
         body: formData
       });
       sentBytes += thisChunkSize;
-      progress(sentBytes / totalBytes);
+      if (typeof options.progress === 'function') {
+        options.progress(sentBytes, totalBytes);
```

Comment:
This is an itty bitty bc break; please check `options.progress.length` and make sure it accepts 2 arguments, othrewise send the previous calculation. (We published this API and others may be using it.)